content/docs/pipelines/sdk/install-sdk.md

```diff
@@ -73,7 +73,22 @@ up Python using [Miniconda](https://conda.io/miniconda.html):
 Run the following command to install the Kubeflow Pipelines SDK:
 
 ```bash
-pip install https://storage.googleapis.com/ml-pipeline/release/latest/kfp.tar.gz --upgrade
+python3 -m pip install kfp --upgrade
```

Comment:
I double checked this and I was incorrect, conda appears to map python and python3 to the same install of Python. So, pip3 install should be safe. Would you be willing to change it back?

pip install kfp --upgrade

Sorry for the confusion on my part.

Reply:
Oops, meant pip3...

pip3 install kfp --upgrade

Reply:
no worries at all. This is quite tricky, across the team we have had many issues with conda , virtual env python3 and pip3 :). 
I agree, python3 -m pip ...  is the safest path, however the advantage is not significant comparing to pip3. I like the pip3 more as it will be more familiar for the users. 
I'm updating to pip3 install kfp --upgrade as per your comment.

Thank you for all the work  and support :) 

Cheers, 
Sina